packages/common/src/restHelpersGet.ts

```diff
@@ -0,0 +1,598 @@
+/** @license
+ * Copyright 2018 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Provides common item fetch functions involving the arcgis-rest-js library.
+ *
+ * @module restHelpers
```

Comment:
? restHelpersGet

Reply:
Split out get functions from restHelpers because file was getting too big. Would like to split out another part--such as updates--someday.

Reply:
I hear you...just wondering if the value  after the module decorator restHelpers should change to restHelpersGet as well. I really don't know exactly what it does but just mentioned it as it was not in sync with the modules name :)

Reply:
You're right--it's an error; will fix